tool/tsh/dbcmd.go

```diff
@@ -315,3 +323,20 @@ func (c *cliCommandBuilder) getMongoCommand() *exec.Cmd {
 	// fall back to `mongo` if `mongosh` isn't found
 	return exec.Command(mongoBin, args...)
 }
+
+func (c *cliCommandBuilder) getSQLServerCommand() *exec.Cmd {
+	args := []string{
+		// Host and port must be comma-separated.
+		"-S", fmt.Sprintf("%v,%v", c.host, c.port),
```

Comment:
I'd assume that mssql will be used also on Windows. Does Windows version also work with `-S`, or only with `\S`? AFAIK some apps works in this way.

Reply:
mssql-cli is a Python program so same flag names should work. I haven't tried it on Windows though, something to try out.